Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs

```diff
@@ -137,18 +137,18 @@ public sealed class InteractiveBrokersBrokerage : Brokerage, IDataQueueHandler,
         public override bool IsConnected => _client != null && _client.Connected && !_stateManager.Disconnected1100Fired;
 
         /// <summary>
-        /// Returns true if the connected user is a financial advisor or non-disclosed broker
+        /// Returns true if the connected user is a financial advisor
         /// </summary>
         public bool IsFinancialAdvisor => IsMasterAccount(_account);
 
         /// <summary>
-        /// Returns true if the account is a financial advisor or non-disclosed broker master account
+        /// Returns true if the account is a financial advisor master account
         /// </summary>
         /// <param name="account">The account code</param>
         /// <returns>True if the account is a master account</returns>
         public static bool IsMasterAccount(string account)
         {
-            return account.Contains("F") || account.Contains("I");
+            return account.Contains("F");
```

Comment:
Mind expanding why is it correct to remove ` || account.Contains("I")`?

Reply:
`Ixxxxxxx` account codes are for Non-Disclosed Introducing Brokers master accounts which are not tradable, according to the IB manual here: https://www.interactivebrokers.com/download/FDBrokersGetStarted.pdf

![image](https://user-images.githubusercontent.com/7236689/105866354-302f5780-5ff4-11eb-96ee-0fb3a1ed097d.png)